tests/ragger/test_eip2930.py

```diff
@@ -0,0 +1,62 @@
+from web3 import Web3
+
+from client.client import EthAppClient
+import client.response_parser as ResponseParser
+from client.utils import recover_transaction
+
+from ragger.backend import BackendInterface
+from ragger.firmware import Firmware
+from ragger.navigator import Navigator, NavInsID
+
+from constants import ROOT_SNAPSHOT_PATH
+
+
+BIP32_PATH = "m/44'/60'/0'/0/0"
+NONCE = 21
+GAS_PRICE = 5
+GAS_LIMIT = 21000
+ADDR = bytes.fromhex("b2bb2b958afa2e96dab3f3ce7162b87daea39017")
+AMOUNT = 0.01
+CHAIN_ID = 3
+
+
+def test_sign_eip_2930(firmware: Firmware,
+                       backend: BackendInterface,
+                       navigator: Navigator,
+                       test_name: str):
+
+    tx_params = {
```

Comment:
Without an `accessList` element, this ends up being a legacy TX.
Add something like this after the `chainId` :
```python
"accessList": [
    {
        "address": "0x0000000000000000000000000000000000000001",
        "storageKeys": [
            "0x0100000000000000000000000000000000000000000000000000000000000000"
        ]
    }
],
```
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2930.md

Reply:
Done